.gitignore

```diff
@@ -5,7 +5,7 @@ __pycache__/
 
 # C extensions
 *.so
-
+.env 
```

Comment:
This can already be found on line 125 under environments (thankfully!) 
https://github.com/autogenai/easy-problems-that-llms-get-wrong/blob/41b335506f3f408f095482bbb569bd2d355a5a47/.gitignore#L125

Reply:
Yes it's already in the .gitingore file.   I was a little overzealous.